src/enqueue.cc

```diff
@@ -99,7 +102,8 @@ ncclResult_t ncclInitKernelsForDevice(int cudaArch, int maxSharedMem, size_t* ma
 
 #ifdef GENERATE_SYM_KERNELS
   for (int sym=0; sym <= 1; sym++) {
-    int kcount = sym==0 ? KernelCount : ncclSymKernelCount;
+    int kcount = sym==0 ? ncclDevKernelCount : ncclSymkKernelCount;
```

Comment:
```suggestion
    int kcount = sym==0 ? KernelCount : ncclSymkKernelCount;
```
RCCL seems to have purposely maintained the use of `KernelCount`. @BertanDogancay do you know why?

Reply:
We should keep `KernelCount` since rccl has its own kernel list defined at the beginning of this file.